### ISSUE TITLE
Molcraft Results and Predicted Affinity Values

### ISSUE THREAD

**Issue by xxrelax** [EDAPINENUT/CBGBench#9]
Dear Author,

I hope this message finds you well.  I am writing to inquire whether the results for Molcraft were obtained by retraining the model or by using the pre-trained checkpoints provided by the Molcraft authors.  Additionally, if possible, I would greatly appreciate it if you could share the predicted affinity values for the reference molecules, as I have noticed some discrepancies between my measured Improvement (%) in Vina Mean values and those reported in your work.

Thank you for your time and consideration.

Best regards,

**Comment by EDAPINENUT** [comment]
We give the selected generated molecules by state-of-the-art methods in our benchmark for the de novo task, which can be downloaded from the [Google Drive](https://drive.google.com/drive/folders/14A_fOf_1088YHedz7Cbe-SIckZ0ltJGQ). You can directly download them for evaluation. Besides, in each directory, there are metrics files ending with `.pt` as the raw evaluation results.

**Comment by xxrelax** [comment]
Thank you for your response. I am still unsure whether the results for Molcraft were generated by retraining the model or using the pre-trained checkpoints provided by the authors.

**Comment by xxrelax** [comment]
Thank you for your response. I am still unsure whether the results for Molcraft were generated by retraining the model or using the checkpoints provided by Molcraft's authors.

**Comment by EDAPINENUT** [comment]
It is generated by the pretrained ckpt in the official repository, since the integration of molcraft is still in process so we did not train it with our framework.

**Comment by xxrelax** [comment]
I used the ligan files provided in your [Google Drive](https://drive.google.com/drive/folders/14A_fOf_1088YHedz7Cbe-SIckZ0ltJGQ) and ran the cal_chem_results.py script. However, the results show significant differences from the "Improvement (%) in Vina Mean" values reported in your paper. Could you kindly advise on what might have caused this discrepancy?
{
    "QED mean": 0.45526483363478926,
    "logP mean": 0.5587105853976541,
    "SA mean": 0.6608865710560626,
    "Lipinski mean": 4.386962190352021,
    "Validity mean": 0.38737373737373737,
    "Score Only": {
        "Improvement (%) in Vina Mean": 2.467608775786709,
        "Delta Binding (%)": 63.85227446111151,
        "Mean Vina": -6.8413954801703625,
        "Ligand Efficiency Mean": -0.34676232210563185
    },
    "Minimize": {
        "Improvement (%) in Vina Mean": 6.628855404698023,
        "Delta Binding (%)": 70.59769034260123,
        "Mean Vina": -7.073308828742405,
        "Ligand Efficiency Mean": -0.36325594772570274
    },
    "Dock": {
        "Improvement (%) in Vina Mean": 4.7251834061504505,
        "Delta Binding (%)": 72.44459575489141,
        "Mean Vina": -7.679493343467656,
        "Ligand Efficiency Mean": -0.39574744227280867
    }
}

**Comment by EDAPINENUT** [comment]
We are sorry that the metric name is different in our initial evaluation scripts and the released ones. 
As you can see from Line. 128 - 147 in https://github.com/EDAPINENUT/CBGBench/blob/master/evaluate_scripts/cal_chem_results.py 
```
 final_results.update({
        "Score Only": {
            "Improvement (MPBG%) in Vina Mean": score_means[0],
            "Delta Binding (IMP%)": score_means[1],
            "Mean Vina (Evina)": score_means[2],
            "Ligand Efficiency Mean (LBE)": score_means[3]
        },
        "Minimize": {
            "Improvement (MPBG%) in Vina Mean": minimize_means[0],
            "Delta Binding (IMP%)": minimize_means[1],
            "Mean Vina (Evina)": minimize_means[2],
            "Ligand Efficiency Mean (LBE)": minimize_means[3]
        },
        "Dock": {
            "Improvement (MPBG%) in Vina Mean": dock_means[0],
            "Delta Binding (IMP%)": dock_means[1],
            "Mean Vina (Evina)": dock_means[2],
            "Ligand Efficiency Mean (LBE)": dock_means[3]
        }
    })
```

In this way, the Improvement is the MPBG, the Delta Binding is the IMP%.  This has been addressed in these days, but the results are consistent with the reported metrics in the paper. If you still have some doubts, you may directly use our scripts to evaluate them, and the newly-released scripts will certainly give the correct results which is the same as the reported one in the paper.

**Comment by xxrelax** [comment]
Thank you for your clarification.  The explanation regarding the metric name discrepancy has resolved my query.

**Comment by xxrelax** [comment]
Could you please provide the generated results files for DiffSBDD, DIFFBP, and FLAG? They would be very helpful for my research.

Thank you for your time and assistance.

**Comment by EDAPINENUT** [comment]
At the time, multiple versions were generated, but the version management is a bit disorganized now. We need to verify them soon before uploading.

**Comment by xxrelax** [comment]
Thank you for your response.  Please take your time to verify the files. Once the files are uploaded, I’d greatly appreciate it if you could let me know. Thank you again for your efforts and support.

**Comment by xxrelax** [comment]
I hope you're doing well. Could you please provide an estimated release time for the remaining generated results files? Your help would be greatly appreciated!

**Comment by EDAPINENUT** [comment]
We plan to spend considerable time on this, as other related matters are currently keeping us busy. We will notify you after we upload it. You can also directly sample from their GitHub repository, as most models have provided checkpoints. Thank you!
